website/content/docs/concepts/filtering/worker-tags.mdx

```diff
@@ -140,13 +140,6 @@ Once workers have tags, you can use these tags to control which
 workers are allowed to manage a given session by specifying optional worker filter attributes
 when you configure [targets](/boundary/docs/concepts/domain-model/targets).
 
-The `egress_worker_filter` attribute controls which workers are used for egress to a target. This is the worker
-that accesses the target.
-
-The `ingress_worker_filter`<sup>HCP Only</sup> attribute controls which workers are used for ingress to a target.
-This is the worker a client connects to when initiating a connection to a target.
-
-
 # Vault Worker Filtering <sup>HCP Only</sup>
 Tags are used to control which [PKI workers] can manage Vault requests by specifying
 a `worker_filter`attribute when configuring [credential stores].
```

Comment:
check my math here, it's only the targets that have a new `egress_worker_filter` attribute, right? I think credential stores still use the `worker_filter` parameter. 

So this should be fine, I just want to make sure I didn't miss something here

Reply:
Good question. Yes, that's correct as far as I understand too. This enhancement shouldn't have affected the credential stores parameter at all. I just rechecked the PRD/RFC and I don't see any reference to an impact to credential stores either.